index.md

```diff
@@ -4,6 +4,74 @@
 The IVOA define a standard URI scheme to reference individual datasets. In this document we will define how these IVOA Identifiers will be formed for Butler datasets and possibly object catalog IDs.
 ```
 
-## Add content here
+## Introduction
 
-See the [Documenteer documentation](https://documenteer.lsst.io/technotes/index.html) for tips on how to write and configure your new technote.
+The IVOA Identifier (IVOID) standard {cite:p}`2016ivoa.spec.0523D` provides a mechanism for publishers to provide unique identifiers to individual datasets using an `ivo` URI scheme.
+The IVOID consists of a registry part that must be registered in VO Registry, and a local part that defines the specific resource.
+The registry part consists of a naming authority (effectively an internet location corresponding to the `netloc` field) and a resource key (corresponding to the `path`).
+The local part corresponds to the `query` component of a URI and can additional be fine-tuned by the use of a fragment.
+This document will address the naming authority, resource key, and query parts of the URI.
+
+## Naming Authority
+
+There is no requirement that the naming authority component correspond to a public web site since the value is looked up in the VO Registry.
+Indeed, IVOA recommendation is to deliberately avoid using a website with examples given such as `ivo://org.gavo.dc/` (reversing the internet address) and `ivo://adil.ncsa` (publishing organization and archive).
+
+This suggests Rubin should consider:
+
+* `lsst.rubin`
+* `rubin.lsst`
+* `org.lsst`
+* `org.rubinobservatory`
+* `org.rubinobs`
+
+If `lsst` is in the authority name that implies we would change the authority name in the future for some potential non-LSST survey in the late 2030s.
+
+## Resource Key
```

Comment:
Agreed today to include the DAC identity as the first /-delimited element of the resource key.  Specifically, `usdac` for the data access services at the US DAC at the USDF.